widgets/toggle/__tests__/toggle-test.js

```diff
@@ -3,11 +3,14 @@
 import React from 'react';
 import expect from 'expect';
 import sinon from 'sinon';
+import jsdom from 'mocha-jsdom';
 
 import toggle from '../toggle';
 import RefinementList from '../../../components/RefinementList';
 
 describe('toggle()', () => {
+  jsdom();
```

Comment:
every test that needs some DOM must call `jsdom()`, what it does is that it injects a jsdom env and remove it when the suite finishes.